cli/src/main/groovy/org/sonatype/nexus/cli/IqPolicyEvaluator.groovy

```diff
@@ -0,0 +1,33 @@
+package org.sonatype.nexus.cli
```

Comment:
Interesting... My initial guess was that you'd be invoking the script directly via command line (i.e. groovy IqPolicyEvaluator.groovy) and use Grapes to grab the dependencies. If maven works easy enough then +1.